### PR TITLE
Enterprise content and continuity fixups

### LINKED ISSUE #1210: FAQ isn't consistent between 5.0/1 and 5.2
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [x] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [ ] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Expected Behavior
FAQ versions are consistent across versions.

## Current Behavior
https://github.com/sensu/sensu-docs/blob/master/content/sensu-go/5.2/getting-started/faq.md#L33 contains the following line missing from 5.0/1:

```
[Contact the Sensu sales team][6] for a personalized demo, and see the [getting started guide][28] for more information.
```

## Possible Solution
Add the following to version 5.0/1:

```
[Contact the Sensu sales team][6] for a personalized demo, and see the [getting started guide][28] for more information.

[28]: ../enterprise
```

### PR DESCRIPTION
**Pull request by apaskulin**
## Description

- Adds command output examples for `sensuctl auth` and `sensuctl license` commands
- Moves the RBAC example in the auth guide to the RBAC reference
- Creates placeholder pages for 5.0 and 5.1
- Updates the 5.0 and 5.1 FAQ to reference enterprise
- sensuctl fixups for format, type names, and links to enterprise commands

To do:

- [x] Change license filenames to `sensu_license.json`
- [x] Update enterprise asset links to /sensu

## Motivation and Context

Closes #1210 
Partially addresses #869

### CODE REVIEW

File: content/sensu-go/5.0/installation/auth.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Authentication"
+description: "In addition to built-in RBAC, Sensu includes enterprise-only support for authentication using a Lightweight Directory Access Protocol (LDAP) provider. Read the guide to configure a provider."
+version: "5.0"
+product: "Sensu Go"
+---
+
+Sensu requires username and password authentication to access the [Sensu dashboard][1], [API][8], and command line tool ([sensuctl][2]).
```
Comment (rgeniesse) [inline content/sensu-go/5.0/installation/auth.md:8]: There is no `[8]` link at the bottom. Maybe needs to be `[8]: ../../api/overview`

File: content/sensu-go/5.0/installation/auth.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+For Sensu's [default user credentials][3] and more information about configuring Sensu role based access control, see the [RBAC reference][4] and [guide to creating users][5].
+
+In addition to built-in RBAC, [enterprise-only][6] support for authentication using an authentication provider is available in Sensu Go 5.2.0 and later.
```
Comment (rgeniesse) [inline content/sensu-go/5.0/installation/auth.md]: Probably correct, but `[6]` is used here and the next link. If right, disregard!

File: content/sensu-go/5.2/reference/license.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "License management"
 linkTitle: "License Management"
-description: "Activate and manage your enterprise license for Sensu Go."
+description: "Sensu Go includes license-activated features designed for enterprises. Activate and manage your enterprise license for with sensuctl and your Sensu account. Read the reference documentation to learn more."
```
Comment (rgeniesse) [inline content/sensu-go/5.2/reference/license.md]:
```suggestion
description: "Sensu Go includes license-activated features designed for enterprises. Activate and manage your enterprise license with sensuctl and your Sensu account. Read the reference documentation to learn more."
```

Also the last sentence feels like there should be a link. Or is that pointing to read the rest of this page?

File: content/sensu-go/5.2/sensuctl/reference.md [hunk 4]
```diff
@@ -264,6 +265,7 @@ Sensuctl provides the following commands to manage Sensu resources.
 - [`sensuctl filter`][16]
 - [`sensuctl handler`][17]
 - [`sensuctl hook`][18]
+- [`sensuctl license](../../reference/enterprise) (enterprise only)
```
Comment (rgeniesse) [inline content/sensu-go/5.2/sensuctl/reference.md]:
```suggestion
- [`sensuctl license`](../../reference/enterprise) (enterprise only)
```

File: content/sensu-go/5.0/installation/auth.md (hunk 1 above)
Comment (rgeniesse) [reply content/sensu-go/5.0/installation/auth.md:8]: Also needs to be added to the other versions.

File: content/sensu-go/5.2/reference/license.md (hunk 3 above)
Comment (apaskulin) [reply content/sensu-go/5.2/reference/license.md]: +1 to your suggestion. For some background, having a call-to-action in these SEO descriptions is considered a best practice (per my conversations with GTM), even though there's no link.

**Pull request merged**